spec/peep_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'peep'
+
+describe Peep do
+
+  it '' do
+  end
```

Comment:
I think you wanted some tests here

Maybe:

- respond_to database columns
- any other behaviour